mk/test.mk

```diff
@@ -1,24 +1,24 @@
 .PHONY: test
 
-.PHONY: test/unit
-test/unit: install ## Dev: run unit tests and exit
+.PHONY: .test/unit
+.test/unit: install
 	@TZ=UTC \
 		FORCE_COLOR=1 \
 		npx vitest \
 			-c vite.config.production.ts \
 			run
 
-.PHONY: test/unit/watch
-test/unit/watch: install ## Dev: run unit tests but watch for changes
+.PHONY: .test/unit/watch
+.test/unit/watch: install
 	@TZ=UTC \
 		FORCE_COLOR=1 \
 		npx vitest \
 			-c vite.config.production.ts \
 
 
-.PHONY: test/e2e
-test/e2e: CYPRESS_SPEC?=**/*.feature
-test/e2e: ## Run browser-based e2e tests against a running GUI, you may want to set KUMA_BASE_URL=http://localhost:8080/gui and KUMA_TEST_BROWSER=chrome
```

Comment:
why are we removing this comment? seems still related

Reply:
We moved it over here

https://github.com/kumahq/kuma-gui/pull/3017/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R56